src/js/tests/newsletter-subscription.test.js

```diff
@@ -91,3 +91,60 @@ test('Happy path', async () => {
     '<b>Thank you. Please check your email to confirm your subscription.</b>',
   );
 });
+
+test('IE11 manual fallback', async () => {
+  // Testing that the overall behaviour still works with IE11, which has
+  // no URLSearchParams or complete FormData implementation
+  // Store original implementation
+  const originalURLSearchParams = URLSearchParams;
+
+  // eslint-disable-next-line no-global-assign
+  URLSearchParams = jest.fn(() => {
```

Comment:
I _think_ this should be safer for mocking globals, as it _shouldn't_ possibly interfere with other test suites
https://stackoverflow.com/a/40460395

Reply:
So you're saying you agree that it's OK, yeah? (We also re-assign the original global later, too)

Reply:
I was trying to recommend a different approach but now that I read more about it I think I was indeed just agreeing and there's no difference in using vs not using the `global` object. So all's well 👍